components/automate-cli/pkg/verifyserver/server/api/v1/systemresources.go

```diff
@@ -0,0 +1,24 @@
+package v1
+
+import (
+	"net/http"
+
+	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/response"
+	"github.com/gofiber/fiber"
+)
+
+func (h *Handler) GetSystemResource(c *fiber.Ctx) {
+
+	nodeType := c.Query("node_type", "")
+	deploymentState := c.Query("deployment_state", "")
+	service, err := h.SystemResourceService.GetSystemResourcesForDeployment(nodeType, deploymentState)
+	if err != nil {
+		h.Logger.Debug("error returned from service ", err)
```

Comment:
If its a error returned from service it should be 500 instead of 400 right?

Reply:
I remove query param validation in service so no error will be returned from service